### ISSUE TITLE
Use creation rules on startup.

### ISSUE THREAD

**Issue by Thyverity** [suchnsuch/Tangent#74]
It would be nice to have the ability to select a creation rule to use on start-up to open a specified page or to create a new one, to facilitate daily note taking or for simply having a startpage. And the option to open a specific folder or query as well.
Optional: schedule to open different pages according to time or weekday.

**Comment by taylorhadden** [comment]
This is an interesting idea! What to do on app start up is pretty straightforward, but what about if you just leave Tangent open forever? Would you expect this "start-up" creation rule to run then as well?

**Comment by Thyverity** [comment]
Now that is a interesting question, in the case you leave it running, it would be more akin to automating page creation.
You probably don't want the creation rule to run while the user is writing, so you would be prompted to postpone while the editor is being used.

**Comment by taylorhadden** [comment]
What's the benefit of having the page be created automatically, but be blank, vs having the page created when you request it? If pages were created automatically, you might have several days where you didn't actually interact with that generated page and now you have blank notes, no?

**Comment by Thyverity** [comment]
You're right, it would be dumb to have a bunch of empty pages, probably better if the page were only to be created with the user's confirmation and only then, but that isn't much different to just scheduling on a agenda, so it is pointless.
Better to stick to on-new-session creation.
